data/remnant/remnant ships.txt

```diff
@@ -1192,6 +1192,17 @@ ship "Smew"
 	description "	Though originally designed as lifeboats, the surprisingly mobile freighter-fighter hybrid has seen a remarkable amount of usage in mining fleets, as a way to supplement the otherwise poor cargo capacity of the Ibis. Because their purpose can vary from fleet to fleet, Smews are traditionally sold with a fair amount of free space: enough for a small weapon, upgraded engines, or additional support systems."
 	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."
 
+ship "Smew" "Smew (Miner)"
+	outfits
+		"Thrasher Cannon"
+
+ship "Smew" "Smew (Support)"
+	outfits
+		"Inhibitor Cannon"	
+
+ship "Smew" "Smew (Runabout)"
+	outfits
+		"Hyperdrive"	
```

Comment:
Variants don't work like that. These will override the outfits of the base models, resulting in unusable ships.
We don't have an `add outfits` option at the moment, so you'll have to redefine the full outfit set.
It also doesn't have enough weapon capacity for an Inhibitor Cannon.

Reply:
lol. Removed the inhibitor variant, and expanded the outfit listing.